lexnlp/extract/common/annotations/address_annotation.py

```
@@ -19,7 +18,7 @@ class AddressAnnotation(TextAnnotation):
 
     def __int__(
```

Pylint (4.0.5)

[error] The special method '__int__' expects 0 param(s), 3 were given

(E0302)